pandas/core/generic.py

```diff
@@ -3137,6 +3137,78 @@ def clip_lower(self, threshold, axis=None):
         subset = self.ge(threshold, axis=axis) | isnull(self)
         return self.where(subset, threshold, axis=axis)
 
```

Comment:
There is already a Series.round, so either integrate that one here, or indeed easier to move to frame